### ISSUE TITLE
java.lang.NoClassDefFoundError: blusunrize/immersiveengineering/common/blocks/BlockIEFluid

### ISSUE THREAD

**Issue by marlonpupo24** [BluSunrize/ImmersiveEngineering#3493]
Hello, I'm having an issue with the mod Immersive Engineering when I try to load my Forge server ( Forge 1.12.2 - 14.23.5.2808 ). When loading it it crashes and gives me the following report:

Minecraft Crash Report

[crash-2019-06-13_12.21.27-server.txt](https://github.com/BluSunrize/ImmersiveEngineering/files/3288887/crash-2019-06-13_12.21.27-server.txt)

This is the only mod that is causing me trouble , I used all the other mods before and they where working fine but I need this mods to fuel the locomotives on the immersive railroading mod. That's why is mandatory for me to have it unless you guys could recommend me another mod that has fuel liquids that works for Immersive railroading as well. As I said before, I'm using version 1.12.2 and all the mods are included in the crash log above.  I hope you guys can help me solve this issue. Thanks for your time.

**Comment by mindforger** [comment]
For future reports lease use pastebin for the crashlog, does this happen without sponge?

**Comment by marlonpupo24** [comment]
You got it sir !! it was the sponge forge the trouble maker. I just deleted it and tried loading it again and it works now. I was so confused because I played with immersive engineering in the past and I wasn't sure why it was failing to load makes so much sense now since I didn't have that sponge mod in my mods folder before and I honestly don't even remember why I put it there ahhaah. Thanks so much, sorry for the mistake there I'll make sure to use pastebin in the future.
